aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+    ![Change Authentication Dialog](./azure-ad-b2c/_static/changeauthdialog.png)
+
+5. Complete the form with the following values:
+    
+    | Setting      | Value  | 
+    | ------------ | ------- | 
+    | **Domain Name** | *\<the domain name of your B2C tenant\>* | 
+    | **Application ID** | *\<paste the Application ID from the clipboard\>* |
+    | **Callback Path** | *\<use the default value\>* |
+    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn` |
+    | **Reset password policy** | `B2C_1_SSPR` |
+    | **Edit profile policy** | *\<leave blank\>* | 
+
+    Select the **Copy** link next to **Reply URI** to copy the Reply URI to the clipboard.  Select **OK** to close the **Change Authentication** dialog, and then select **OK** to create the web app.
+
+## Finish the B2C application registration
+
+In the browser window, with the B2C application properties still open, change the temporary **Reply URL** specified earlier to the value copied from Visual Studio, and then select **Save** at the top of the window.
+
+> [!TIP]
+> If you didn't copy the Reply URL, use the SSL address from the Debug tab in the web project properties, and append the **CallbackPath** value from `appsettings.json`.
+
+## Configure policies
+
+Use the steps in the AAD B2C documentation to [create a sign-up or sign-in policy](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-sign-up-or-sign-in-policy), and then [create a password reset policy](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-reference-policies#create-a-password-reset-policy).  Use the example values provided in the documentation for **Identity providers**, **Sign-up attributes**, and **Application claims**.  Using the **Run now** button to test the policies as described in the documentation is optional.
+
+> [!WARNING]
+> Ensure the policy names are exactly as described in the documentation, as those policies were used in the **Change Authentication** dialog in Visual Studio. The policy names can be verified in `appsettings.json`.
+
+## Run the application
+
+In Visual Studio, press **F5** to build and run the application. After the web app launches, select **Sign in**.
+
+![Sign into the application](./azure-ad-b2c/_static/signin.png)
+
+The browser redirects to the AAD B2C tenant.  Sign-in with an existing account (if one was created testing the policies) or select **Sign up now** to create a new account.  The **Forgot your password?** link can be used to reset a forgotten password.
+
+![AAD B2C login](./azure-ad-b2c/_static/b2csts.png)
+
+After successfully signing in, the browser redirects to the web app.
+
+![Success](./azure-ad-b2c/_static/success.png)
+
+## Next steps
+
+Now that the ASP.NET Core application is configured to use Azure Active Directory B2C for authentication, the [Authorize attribute](../../authorization/simple) can be used to secure your application.  Other tasks that may be performed include:
```

Comment:
* Azure Active Directory B2C --> AAD B2C
* Prefer the xref style of linking to external docs